chore(docs): add python api protection code (#224)
logto-io/docs: docs/docs/recipes/protect-your-api/fragments/_token_extract.mdx

```

</TabItem>
<TabItem value="python" label="Python">

> For python developers, there is a handy library [authlib](https://github.com/lepture/authlib) that can easily helps you to validate the breaer token issued by Logto.

Reviewer: ```suggestion
For python developers, there is a handy library [authlib](https://github.com/lepture/authlib) that can easily help you validate the bearer token issued by Logto.
```

Author: updated

Create a resource protector for your private APIs:

:::note
For Flask Developers please refer to the [flask_oauth resource_protector](https://docs.authlib.org/en/latest/flask/2/resource-server.html).
For Django Developers please check [django resource_protector](https://docs.authlib.org/en/latest/django/2/resource-server.html).
:::

```python
from flask import Flask

# For django developers:
from authlib.integrations.django_oauth2 import ResourceProtector

# For Flask Developers
from authlib.integrations.flask_oauth2 import ResourceProtector

> # We will get back to this validator customization on the following validation section

Reviewer: ```suggestion
# We will get back to this validator customization in the following validation section
```

Author: updated

from validator import MyLogtoBearerTokenValidator

> APP = Flask(__name__)

Reviewer 2: Is `APP` by design? or just
```suggestion
App = Flask(__name__)
```

Author: Shouldn't matter. I'll update

require_auth = ResourceProtector()
require_oauth.register_token_validator(MyLogtoBearerTokenValidator())

@app.route("/api/foo")
@require_auth()

> def foo() # your private API here

Reviewer: ```suggestion
def foo() # your protected API here
```

Author: updated
